.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
You may want to add this file to .gitignore moving forward as I don't think it's needed on the repo